evm-adapters/src/sputnik/cheatcodes/cheatcode_handler.rs

```diff
@@ -93,7 +93,7 @@ pub(crate) fn convert_log(log: Log) -> Option<String> {
             format!(
                 "{}: {:?}",
                 inner.key,
-                ethers::utils::parse_units(inner.val, inner.decimals.as_u32()).unwrap()
+                ethers::utils::format_units(inner.val, inner.decimals.as_u32()).unwrap()
```

Comment:
We also need this same change a few lines up for `LogNamedDecimalIntFilter` right?

Reply:
Yeah, although that may have to be an ethers-rs fix, getting this error when changing `parse_units` to `format_units`

```
ethers::utils::format_units(inner.val, inner.decimals.as_u32()).unwrap()
   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `From<ethers::prelude::I256>` is not implemented for `ethers::prelude::U256`
 ```

Reply:
Yeah we do not support I256 for format/parse units yet. 

You could do this here by getting the absolute value of the I256, formatting it as string, and then prepending a minus if its sign is a negative number.

Reply:
Adding support for I256 to ethers-rs is probably the best way to do this right? If so we can merge this PR, and I (or @nanexcool if you'd rather do it) can take a shot at adding I256 support to ethers-rs `format_units`, then follow up with a PR here to change `LogNamedDecimalIntFilter` to use `format_units`

Reply:
@mds1 I think I'm gonna do a couple of Rust tutorials first instead of heading straight for another PR 😂  Go for it!

Reply:
sounds good! I've been working my way through [rustlings](https://github.com/rust-lang/rustlings) which has been pretty good, but please share any good rust tutorials you find! 😛